packages/transmute-framework/package.json

```diff
@@ -38,9 +38,12 @@
     "codecov": "^3.0.0",
     "coveralls": "^3.0.0",
     "docdash": "^0.4.0",
+    "firebase-admin": "^6.0.0",
     "jsdoc": "^3.5.5",
     "react-scripts": "^1.1.1",
-    "solidity-coverage": "^0.4.14"
+    "solidity-coverage": "^0.4.14",
+    "transmute-adapter-ipfs": "1.0.3",
+    "transmute-adapter-firestore": "0.1.0"
```

Comment:
Lets run `npm i` in `production` more, just to make sure we'll have all the packages we need on prod env.

Reply:
As I attempted to do this, i realized react scripts is being used here... also I'm not sure how to do this correctly.

Here is an interesting issue about lerna in production mode...

https://github.com/lerna/lerna/issues/721

installing the framework in production mode would prevent us from running tests.

I'm not sure the value of `production` mode npm installs for frameworks, can you explain further?

Reply:
I'd like to address these concerns in a follow up PR. I've created a ticket to ensure we do.